packages/inter-protocol/test/vaultFactory/vaultFactoryUtils.js

```diff
@@ -127,7 +143,12 @@ export const setupElectorateReserveAndAuction = async (
   };
 
   await startAuctioneer(space, { auctionParams });
-  return { space };
+  return {
+    space,
+    priceAuthority: priceAuthorityReg,
+    priceAuthorityAdmin,
+    aethPriceAuthority,
```

Comment:
same issue here, right? 
```suggestion
    aethManualPA,
```

Reply:
done